web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -216,6 +222,14 @@ export class AccountService {
         account.transactions = transactionList.transactions;
 
         const unconfirmedTransactionsResponse = await this.getUnconfirmedTransactions(account.account);
+        
+        // @ts-ignore - Send notifications for new transactions
+        if (window.Notification) {
+          unconfirmedTransactionsResponse.unconfirmedTransactions
+            .filter(({transaction}) => this.isNewTransaction(transaction))
+            .map((transaction) => this.sendNewTransactionNotification(transaction));
```

Comment:
```suggestion
            .map(this.sendNewTransactionNotification);
```
I think that should work, maybe there could be a binding issue due to subscribe